index.html

```diff
@@ -442,30 +442,17 @@ <h2>User Interface Language</h2>
         </p>
         <p>
           Different jurisdictions have different prerequisites before a platform can enable a universal
-          opt-out. For example, the most recent regulations promulgated under the California Consumer
-          Privacy Act state:
+          opt-out. Many states say that a user agent may not send a universal opt-out signal by "default,"
+          though at least one state has said that selecting a privacy focused user agent is a sufficient
+          indicator of user intent.
         </p>
-        <blockquote cite=https://cppa.ca.gov/regulations/pdf/cppa_regs.pdf>
-          The platform, technology, or mechanism that sends the opt-out preference signal shall make
-          clear to the consumer, whether in its configuration or in disclosures to the public, that
-          the use of the signal is meant to have the effect of opting the consumer out of the sale
-          and sharing of their personal information. The configuration or disclosure does not need to
-          be tailored only to California or to refer to California ([[?CPPA-REGULATIONS]], §7025(b)(2)).
-        </blockquote>
         <p>
-          Colorado and other jurisdictions are more prescriptive about requirements for a valid universal
-          opt-out signal. For example, Colorado’s regulations explicitly provide “a Universal Opt-Out
-          Mechanism may not be the default setting for a tool that comes pre-installed with a device,
-          such as a browser or operating system” ([[?COLORADO-REGULATIONS]], Rule 5.04(a)).
+          Different jurisdiction may also have different rules for when companies can override or disregard
+          a universally applicable opt-out signal, either because they have consent from the user to do so
+          or otherwise.
```

Comment:
Wording nit, since "or otherwise" doesn't add any information:
```suggestion
          a universally applicable opt-out signal,
          for example because they have consent from the user to do so.
```

Reply:
I think there are other scenarios besides consent where a jurisdiction might say you can disregard a generally applicable signal. For example, the law may say it doesn't apply to certain controllers (e.g., small businesses below a certain threshold, nonprofits, &c).

Reply:
I think the "for example" would tell readers that there might be other scenarios. It's better than "either" because "either" implies that there are exactly two situations, and that the second one will be described, which it's not here.